python/dgl/data/dgl_dataset.py

```diff
@@ -0,0 +1,157 @@
+"""Basic DGL Dataset
+"""
+
+from __future__ import absolute_import
+
+import os, sys
+from .utils import download, extract_archive, get_download_dir, makedirs
+from ..utils import retry_method_with_fix
```

Comment:
I don't see anywhere other than `dgl.data` using `retry_method_with_fix`. Do we consider move it into `dgl.data`?

Reply:
You can do it in the next PR.